scripts/run_docker.sh

```diff
@@ -8,6 +8,27 @@ if [[ $(git rev-parse --show-toplevel) != $(pwd) ]]; then
   exit 1;
 fi
 
+if [[ ! -f .env ]]; then
+  (
+  cd web/backend
+  npm ci
```

Comment:
this seems to be erroneous to me

Reply:
ah, yes, that makes sense - than I veto the change, since the whole point of having a dockerized environment is to not run npm locally at all if one so wishes

putting it into a separate script is OK

Reply:
OK, added a `.env.example`